guides/FFI-Tips.md

```diff
@@ -117,7 +117,7 @@ By moving the `show` reference out to `showSomething` the compiler will pick the
 
 ## Why Doesn't my `Eff` Work When Passed to a Normal JS Function?
 
-["Representing Side Effects"](https://leanpub.com/purescript/read#leanpub-auto-representing-side-effects) in *PureScript by Example*.
+["Representing Side Effects"](https://book.purescript.org/chapter8.html#the-effect-monad-1) in *PureScript by Example*.
```

Comment:
I wasn't 100% sure to which section in this chapter I should link. This was my best guess.

Reply:
I think https://book.purescript.org/chapter10.html#representing-side-effects would be more appropriate, as that's the one which deals with the runtime representation of `Effect`.

Reply:
done!